src/docs/product/integrations/azure-devops/index.mdx

```diff
@@ -14,20 +14,15 @@ This integration needs to set up only once per organization, then it is availabl
 ## Install
 
 <Alert title="Note" level="info">
-  Sentry owner or manager permissions, and Azure project admin permissions are
+  Sentry organization owner permissions and Azure project admin permissions are
```

Comment:
this should be organization owner or manager for Sentry. Organization owner is the role the user must have on Azure DevOps.